template/deploy/helm/[[operator]]/templates/deployment.yaml.j2

```diff
@@ -43,11 +43,20 @@ spec:
             - mountPath: /etc/stackable/{{ include "operator.appname" . }}/config-spec
               name: config-spec
           env:
+            # The following env vars are passed as clap (think CLI) arguments to the operator.
+            # They are picked up by clap using the structs defied in
+            # https://github.com/stackabletech/operator-rs/blob/main/crates/stackable-operator/src/cli.rs
+            # (currently "ProductOperatorRun").
+            # You can read there about the expected values and purposes.
```

Comment:
IMO referencing to some obscure Rust code is no good. Maybe I'm blind, but I didn't find what `OPERATOR_IMAGE` is actually needed for and I really don't know.

Reply:
https://github.com/stackabletech/opa-operator/blob/a5a418c585de8a8ee7efa18de5cf243431cac872/rust/operator-binary/src/main.rs#L50-L52

I updated the comment that the struct is actuality defined in the operator, with 90% of the envs being defined in op-rs. The previous comment was a bit misleading.
I also added a description to OPERATOR_IMAGE in e6771ad